manifests/charts/istio-operator/templates/deployment.yaml

```diff
@@ -3,6 +3,13 @@ kind: Deployment
 metadata:
   namespace: {{.Release.Namespace}}
   name: istio-operator{{- if not (eq .Values.revision "") }}-{{ .Values.revision }}{{- end }}
+  labels:
+    istio.io/rev: {{ .Values.revision | default "default" }}
+    operator.istio.io/component: "IstioOperator"
+    release: {{ .Release.Name }}
+    {{- range $key, $val := .Values.Labels }}
+    {{ $key }}: "{{ $val }}"
```

Comment:
I know it has been copied from the template labels but I think the quotation marks are going to be set (if necessary) in the values file so no need to add them again here.
wdyt?

Reply:
Updated